third_party_licenses_test.go

```
@@ -0,0 +1,44 @@
+			t.Fatalf("ReadFile(%s %s): %v", dependency.module, dependency.licenseFile, err)
+		}
+		if !strings.Contains(string(notice), string(license)) {
+			t.Errorf("NOTICE does not include the complete license text for %s", dependency.module)
+		}
+	}
+}
+
+func moduleDir(t *testing.T, module string) string {
+	t.Helper()
+
+	output, err := exec.Command("go", "list", "-m", "-f", "{{.Dir}}", module).Output()
```

golangci-lint (2.13.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)